rust/src/cli/persist_nic.rs

```diff
@@ -52,108 +60,15 @@ fn gather_state() -> Result<NetworkState, CliError> {
     Ok(state)
 }
 
-fn device_is_virtual(iface: &Interface) -> bool {
```

Comment:
Don't we still want this?  On an OCP node I didn't want to persist any of the dynamic ovs interfaces I think...

Reply:
OVS internal interface is not ethernet, so I think we are good here. For OVS system interface, I think there database still want the interface name been persistent.